### ISSUE TITLE
negative durations

### ISSUE THREAD

**Issue by chesles** [medikoo/duration#1]
Durations where `from` is after `to` aren't handled properly:

```
var Duration = require('duration')

var start = new Date(2012, 10, 23, 10);
var end = new Date(2012, 10, 23, 9);

var dur = new Duration(start, end);
console.log(dur.toString());
// => "-1y -1m -1d -1:00:00.000"
```

If I specify a format string, it seems to do OK, for the most part:

```
dur.toString("%Hs"); // => "-1"
dur.toString("%Ms"); // => "-60"
dur(s, e).toString("%ds") // => "-1" (should be 0)
```

Other than this little bug, `duration` is great. Thanks!

**Comment by krazyjakee** [comment]
@medikoo I think this needs to be reopened: https://repl.it/repls/ExoticBeautifulBrackets

**Comment by medikoo** [comment]
> @medikoo I think this needs to be reopened: https://repl.it/repls/ExoticBeautifulBrackets

Fix published with v0.2.2
